analytics/terraform/spark-k8s-operator/examples/s3-tables/README.md

```diff
@@ -53,6 +53,39 @@ aws s3tables create-table-bucket \
 
 Make note of the S3TABLE ARN generated by this command.
 
+<details>
```

Comment:
Let's move this doc under Website docs. Here is the link https://github.com/awslabs/data-on-eks/blob/main/website/docs/blueprints/data-analytics/spark-operator-s3tables.md

We can add a new section called "**S3 Tables with Jupyterhub on EKS**"